src/System.Reflection/tests/Assembly/Assembly_EmbeddedFiles.cs

```diff
@@ -16,6 +16,10 @@ public class EmbeddedFilesTest
         [Fact]
         public void TestEmbeddedFiles()
         {
+            var resources = typeof(EmbeddedFilesTest).GetTypeInfo().Assembly.GetManifestResourceNames();
+
+            Assert.Contains("EmbeddedImage.png", resources);
+
```

Comment:
Reading the stream of an embedded resource is getting verified in one of the tests I wrote earlier - https://github.com/dotnet/corefx/blob/master/src/System.Runtime.Loader/tests/ResourceAssemblyLoadContext.cs#L30